config.yml.template

```diff
@@ -1,10 +1,9 @@
 email:
   username: my_email@gmail.com
   # for Gmail, password should be an application-specific password
+  # App Passwords are depeciated.  Follow Readme to set up OAuth.
+  # password line can be removed, if desired.
```

Comment:
let's specify that this can be removed iff you've set up OAuth

Reply:
fixed